configure

```diff
@@ -264,9 +264,16 @@ sub parseArgs {
 my $cross_compile = 0;
 my %opts = %{parseArgs()};
 
+my $debug = exists $opts{'debug'} ? $opts{'debug'} : 1;
+if ($opts{'verbose'}) {
```

Comment:
Is this level of detail needed in the verbose log output?  It already dumped out the `%opts`.

Reply:
No, I think I was using it to test the debug variable above.